recipe/meta.yaml

```diff
@@ -20,6 +20,8 @@ requirements:
     - pip
     - setuptools
     - wheel
+    - hatchling
+    - hatch-vcs
```

Comment:
- I'm not sure why `setuptools` and `wheel` are there, they are not in the conda-forge recipe.
- I've added `hatchling` and `hatch-vsc` as in the conda-forge recipe, they are indeed new build dependencies https://github.com/holoviz/param/blob/b6a285b7205a869b5677b8571fd54c2a5ccc8709/pyproject.toml#L1-L3

Reply:
- They switched from `setuptools` to `hatch`: https://github.com/holoviz/param/commit/aa826ef429bc13f34e0179135e160a0a022faf39
- we can remove `setuptools` and `wheel` probably

Reply:
@lorepirri do you want me to remove `setuptools` and `wheel`?

Reply:
@maximlt, yes, in the commit Lorezno mentioned, there is:
```
- python -m pip install setuptools wheel twine tox
+ python -m pip install build
```

Reply:
If also @M-Waszkiewicz-Anaconda agrees with the intuition, I would remove them. `hatch` is another build system, and I don't see `setuptools` being used anywhere. There is a reference for an optional `setuptools_scm` though.

Reply:
Removed in https://github.com/AnacondaRecipes/param-feedstock/pull/7/commits/297b6053912b5ee05e5040144dd259d52fa69521.

> There is a reference for an optional setuptools_scm though.

`setuptools_scm` is used to deal with editable installs of the package, so I think it's not relevant here.

Reply:
Thanks both for your answers!

Reply:
ok perfect!